marathon_envoy_poc/envoy.py

```diff
@@ -150,3 +152,127 @@ def ClusterLoadAssignment(cluster_name, lb_endpoints):
             "drop_overload": 0.0,
         },
     }
+
+
+def Listener(name, address, port, filter_chains=[]):
+    # https://www.envoyproxy.io/docs/envoy/v1.5.0/api-v2/lds.proto#listener
+    return {
+        "name": "http",
+        "address": Address(address, port),
+        "filter_chains": filter_chains,
+        # "use_original_dst": "{...}",
+        # "per_connection_buffer_limit_bytes": "{...}",
+        # "drain_type": "..."
+    }
+
+
+def FilterChain(filters, sni_domains=""):
```

Comment:
`sni_domains` is a `repeated string` which, as far as I can tell, means it should be a list of strings rather than a single string.

Reply:
Thanks.. Envoy _seems_ to accept single strings as well as arrays of them.. but I will change this.